website/blog/2020-06-23-deliver-faster-applications-with-wavemaker.md

```diff
@@ -0,0 +1,69 @@
+
+### Content Delivery Network
+A Content Delivery Network(CDN) is a set of the geographically distributed servers which together enable fast, reliable & secure delivery of internet content including HTML, Javascript, Stylesheets, Images, etc., 
+
+With CDN, the content is served from the CDN-edge server closest to the user location helping in faster load times. With caching & other optimizations CDN also reduces the amount of data origin servers must provide, thus reducing hosting costs. With its distributed nature CDN can ensure better availability and can handle failures better than any origin servers. With all these benefits CDN ensures consistent user experience for an app across the geographical locations. 
+
+To enable an application to make its artifacts served over CDN, the developer needs to handle additional operations such as,
+* Configure the Origin source & Content Delivery Network
+* Managing build artifacts with each release
+* Cache invalidations to ensure the user is served latest response
+* Additional operations based on CDN provider & more
+
+But we wanted our low code developer to have access to all the CDN goodness very easily. So WaveMaker has made it available on click of a button, literally. With the current release, when the user deploys the WaveMaker app by clicking the "Deploy" button, it is deployed on AWS with CDN support without any additional configuration. This convenience is only available in the first phase of the deployment pipelines of the application. Note that, applications are deployed to this phase so that the app can be tested by the team & only last for limited period of time. [See deploying to live phase](https://www.wavemaker.com/learn/app-development/deployment/manage-deployed-apps#configure-live-phase).
+
+### CDN intergration with WaveMaker
+
+Under the hood, WaveMaker handles all operations required for CDN deployment of apps as listed below,
+* Build process is updated with artifacts generation for front-end only static assets such as javascript, CSS, etc.,
+* Upload the static assets of the project to Amazon S3
+* CloudFront which is AWS-CDN service provider configured with all the required properties.
+* Updates the deployed project to serve static assets from Cloudfront CDN.
+
+Thus ensuring the customers benefit from CDN advantages for every app developed with no additional effort.
+
+However, for deploying onto production, the customers may prefer their cloud service provider with the help of CI, CD pipelines. The documentation for which will be available soon. In the subsequent releases, we will also support one-click deployment to deploy on customer cloud providers. 
+
+### Results
+
+Below are the stats from WaveMaker testing with CDN integration comparing the download times of static assets for some of the standard sized applications,
+
+| Application | Reduced Download Time |
+|-------------|-----------------------|
+| Starter App |                   ~39%|
```

Comment:
What is this starter app? Can we put numbers for the other showcase apps instead of the starter app?

Reply:
Sure...